lib/Qandidate/Toggle/Serializer/ContextSerializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Qandidate\Toggle\Serializer;
+
+use Qandidate\Toggle\Context;
+
+class ContextSerializer
+{
+    /**
+     * @param array $data
+     *
+     * @return Context
+     */
+    public function deserialize(array $data)
+    {
+        $this->assertHasKey('context', $data);
+        if ( ! is_array($data['context'])) {
+            throw new \RuntimeException('Key "context" should be an array.');
+        }
+
+        $context = new Context();
+        foreach ($data['context'] as $key => $value) {
+            if (!$context->has($key)) {
+                $context->set($key, $value);
+            }
+        }
+
+        return $context;
+    }
```

Comment:
As I see this, the serializers are only useful when using the lib through the API, and a `serialize` method would only make sense if the API itself would send back the context in the response. But I couldn't think of any case where it might be needed.